client/dashboard/app/queries/site.ts

```diff
@@ -26,6 +26,23 @@ export const siteBySlugQuery = ( siteSlug: string ) => ( {
 } );
 
 export const siteByIdQuery = ( siteId: number ) => ( {
+	queryKey: [ 'site-by-slug', siteId, SITE_FIELDS, SITE_OPTIONS ],
```

Comment:
Should this be `site-by-id`?

Reply:
Done by https://github.com/Automattic/wp-calypso/pull/104710/commits/6a47d6e9cce295797873169074e1e9acf1097049.